Packs/Kenna/Integrations/KennaV2/KennaV2.py

```diff
@@ -339,10 +424,10 @@ def search_assets(client: Client, args: dict) -> Tuple[str, Dict[str, Any], List
     else:
         tags = args.get('tags')
     params = {
-        'id[]': argToList(args.get('id')),
-        'hostname[]': argToList(args.get('hostname')),
+        'id': argToList(args.get('id')),
+        'hostname': argToList(args.get('hostname')),
         'min_risk_meter_score': args.get('min-score'),
-        'tags[]': tags
+        'tags': tags
```

Comment:
Can you please explain why you changed these lines?

Reply:
Hey Bar, I may have been working from an old copy of the integration, when I had started coding.  Not really sure which version of the line is correct.  It was not something that I had included in my changes.